apps/frontend/src/pages/Build/BuildInfoDialog.tsx

```diff
@@ -0,0 +1,100 @@
+import { useParams } from "react-router-dom";
+import { BuildType } from "@/gql/graphql";
+
+import {
+  Dialog,
+  DialogBody,
+  DialogDismiss,
+  DialogFooter,
+  DialogText,
+  DialogTitle,
+  useDialogState,
+} from "@/ui/Dialog";
+import { Code } from "@/ui/Code";
+import { Link } from "@/ui/Link";
+
+const useDialogContent = (
+  buildType: BuildType | null | undefined,
+  referenceBranch: string | null | undefined,
+) => {
+  const { accountSlug, projectName } = useParams();
```

Comment:
`useParams` is not good here, you should pass params to the dialog itself, if we want to reuse it elsewhere it's better.